src/Common/RJSF/templates/FieldErrorTemplate.tsx

```diff
@@ -24,14 +24,17 @@ export const FieldErrorTemplate = ({ errors = [], idSchema }: FieldErrorProps) =
 
     return (
         errors.length > 0 && (
-            <span id={id}>
+            <span className="display-grid rjsf-form-template__field dc__gap-12" id={id}>
```

Comment:
instead of using the field class. create a create field-error

Reply:
I want the error to align with the text input of fields. That's why reusing the same grid layout

Reply:
Add a class using ,